### PR TITLE
EOS-16355: Message Bus - Remove rabbitmq refrences

### PR DESCRIPTION
**Pull request by sandeepsgt**
Signed-off-by: Sandeep Anjara <sandeep.anjara@seagate.com>

## Problem Statement
<pre>
  <code>
    Story Ref (if any):
    Please add Problem statement here...
  </code>
</pre>
## Problem Description
<pre>
  <code>
    Please add Problem decription here...
  </code>
</pre>
## Solution
<pre>
  <code>
    Please add summary of the change,List any dependencies that are required for this change.
  </code>
</pre>
## Sanity testing on RPM done
<pre>
  <code>
    - [ ] Yes
    - [ ] No
  </code>
</pre>
## Unit/Manual Testing Description
<pre>
  <code>
    Please describe the tests that you ran to verify your changes.
  </code>
</pre>

### CODE REVIEW

**Review by thavanathan — APPROVED** [review]

File: low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R1.yaml [hunk 1]
```diff
@@ -23,43 +23,28 @@ SYSTEM_INFORMATION:
...
-    password:
-    primary_rabbitmq_host: localhost
-
-RABBITMQCLUSTER:
+   consumer_id: sspl_in
+   consumer_group: cortx_monitor
+   message_type: IEM
+   offset: earliest
+
+MESSAGINGCLUSTER:
```
[the remark this answers is not in the document]
Comment (sandeepsgt) [reply low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R1.yaml]: Not needed, Removed

File: low-level/files/opt/seagate/sspl/conf/sspl.conf.sample.yaml [hunk 2]
```diff
@@ -91,41 +91,26 @@ SYSTEM_INFORMATION:
...
-RABBITMQEGRESSPROCESSOR:
-   virtual_host: SSPL
-   queue_name: sensor-queue
-   exchange_name: sspl-out
-   routing_key: sensor-key
-   ack_queue_name: sensor-queue
-   ack_exchange_name: sspl-out
-   ack_routing_key: sensor-key
-   username: sspluser
-   password: gAAAAABf_F2Xh3y0x2rPTRGL0ld4f_u1AwI5pRHOOEk49-lSNsq339Mn8m9bSBsGH-U8bCaPnTJPW-LR4yQHc_CpSbhU3AL5wQ==
```
[the remark this answers is not in the document]
Comment (sandeepsgt) [reply low-level/files/opt/seagate/sspl/conf/sspl.conf.sample.yaml:112]: `Do we have equivalent in message bus (kafka)?` No, hence removed

File: low-level/framework/messaging/plane_cntrl_rmq_ingress_processor.py [hunk 3]
```diff
@@ -35,7 +35,7 @@
 from framework.base.internal_msgQ import InternalMsgQ
```
[the remark this answers is not in the document]
Comment (sandeepsgt) [reply low-level/framework/messaging/plane_cntrl_rmq_ingress_processor.py]: Removed  plane_cntrl_rmq_*_processor files

File: low-level/framework/messaging/rabbitmq_connector.py [hunk 4]
```diff
@@ -28,7 +28,7 @@
 from framework.utils.conf_utils import SSPL_CONF, Conf
 
 
-RABBITMQ_CLUSTER_SECTION = 'RABBITMQCLUSTER'
+RABBITMQ_CLUSTER_SECTION = 'MESSAGINGCLUSTER'
```
[the remark this answers is not in the document]
Comment (sandeepsgt) [reply low-level/framework/messaging/rabbitmq_connector.py]: rabbitmq_connector.py not needed now, if at all we have to use rabbitmq for some reason, message bus will provide the interface

File: low-level/framework/sspl_rabbitmq_reinit.py [hunk 5]
```diff
@@ -37,9 +37,9 @@
 from cortx.utils.conf_store import Conf
 
 RABBITMQCTL = '/usr/sbin/rabbitmqctl'
-SECTION="RABBITMQEGRESSPROCESSOR"
-INGRESS_CONFIG_SECTION = 'RABBITMQINGRESSPROCESSOR'
-EGRESS_CONFIG_SECTION = 'RABBITMQEGRESSPROCESSOR'
+SECTION="EGRESSPROCESSOR"
```
[the remark this answers is not in the document]
Comment (sandeepsgt) [reply low-level/framework/sspl_rabbitmq_reinit.py:40]: Not needed, removed

**Review by cdeshmukh — APPROVED** [review]

**Pull request merged**